src/components/Popover/src/Popover.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+	<div style="display: contents;">
+		<slot
```

Comment:
The tether API feels like an additional layer of configuration that isn't needed right now. By restricting the current use-cases to specific elements in the same view we can reduce the complexity.

Are there current use cases we need to consider to wrap other elements?

Reply:
The tether slot here is the same as the action slot in orbit. I actually removed the external tether api that I mentioned in the issue.

Reply:
I kept it as 'tether' because this version allows a parent component to trigger the open/close of the popover, so there may not actually be an action inside the slot

Reply:
I see now, just nomenclature change. I had forgotten about that detail in Orbit. But now I'm trying to remember why we needed it to begin with since at first glance the default slot and scoped slot would work the same way.